src/content/docs/mlops/integrations/mona-mlops-integration.mdx

```diff
@@ -19,14 +15,14 @@ import mona03 from 'images/mona03.png'
 
 Mona collects and analyzes data from your entire AI system in production, allowing you to evaluate model quality with both out-of-the-box and custom metrics. It's able to detect data-drift, model-bias or data integrity issues via actionable insights, even if those occur only in specific segments of your data.
 
-Mona now offers an [MLOps](/docs/alerts-applied-intelligence/mlops/get-started/intro-mlops/) integration with New Relic so that you can add the state of your AI as monitored by Mona to your centralized operational view on New Relic.
+Mona includes an [model performance monitoring](/docs/alerts-applied-intelligence/mlops/get-started/intro-mlops/) integration with New Relic so that you can add the state of your AI as monitored by Mona to your centralized operational view on New Relic.
```

Comment:
```suggestion
Mona includes a [model performance monitoring](/docs/alerts-applied-intelligence/mlops/get-started/intro-mlops/) integration with New Relic so that you can add the state of your AI as monitored by Mona to your centralized operational view on New Relic.
```